xarray/core/variable.py

```diff
@@ -2528,7 +2532,7 @@ def _to_dense(self) -> Variable:
 
     def chunk(  # type: ignore[override]
         self,
-        chunks: T_Chunks = {},
+        chunks: T_Chunks = None,
```

Comment:
IIRC these are intended to be different between `{}` and `None` (and the coercion below then doesn't let `None` through...)

Reply:
...though the tests pass, so maybe it's fine (either that or our tests aren't sufficient...)

Reply:
Yes, there was a couple of instances where a comment was explaining something like this, so I left these ones alone.

https://github.com/pydata/xarray/blob/1c6300c415efebac15f5ee668a3ef6419dbeab63/xarray/core/dataset.py#L2660

https://github.com/pydata/xarray/blob/1c6300c415efebac15f5ee668a3ef6419dbeab63/xarray/core/dataarray.py#L1363

In the absence of such comment (like here) I replaced the default with `None`, but I could also revert these places and add a comment if it was missing.

Reply:
Yes that would be great.

Reply:
@dcherian to confirm, you would like this:

>  I could also revert these places and add a comment if it was missing.

Reply:
^ yes.